plugins/GSdx/GSRendererOGL.cpp

```diff
@@ -606,6 +597,32 @@ void GSRendererOGL::SendDraw(bool require_barrier)
 		ASSERT(GLLoader::found_GL_ARB_texture_barrier);
 		glTextureBarrier();
 		dev->DrawIndexedPrimitive();
+	} else if (m_vt.m_primclass == GS_SPRITE_CLASS) {
+		size_t nb_vertex = (GLLoader::found_geometry_shader) ? 2 : 6;
+
+		GL_PUSH("Split the draw");
+
+#if defined(ENABLE_OGL_DEBUG)
```

Comment:
Use _DEBUG here instead. OGL_DEBUG is enabled on dev build and it is already slow enough.